testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java

```diff
@@ -84,6 +84,66 @@ public void testFailingHiveViewsForInformationSchema()
                 .hasMessageContaining("Failed to translate Hive view 'test_list_failing_views.failing_view'");
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testFailingHiveViewsForSystemJdbc()
```

Comment:
can we merge this with above test to avoid duplicating idiosyncratic testcases and comments?

Reply:
I would rather not do that. The structure is very similar but queries are different (and not only on the queried table) and some assertions too (e.g `system.jdbc.tables` returns broken views while `information_schema.views` does not). 
I think if I merge the tests it will be less readable.

Reply:
My intuition would also be to keep the test scenario-centric.

We should think of `system.jdbc.{tables,columns}` as unfortunately two parallel implementations of the same thing. Testing them in tandem indeed adds more text in the test, but also ensures coherency (test covered together). FWIW We do test them together for materialized views.


if test is getting too long we can structure it

```
setup

verifyInformationSchemaForBrokenViews();
verifyJdbcTablesForBrokenViews();

cleanup
```

Reply:
I extracted `setupBrokenView`